client/my-sites/theme/main.jsx

```diff
@@ -273,7 +282,12 @@ const ThemeSheet = React.createClass( {
 					{ i18n.translate( 'Need extra help?' ) }
 					<small>{ i18n.translate( 'Get in touch with our support team' ) }</small>
 				</div>
-				<Button primary={ isPrimary } href={ '/help/contact/' }>Contact us</Button>
+				<Button
+					primary={ isPrimary }
+					href={ '/help/contact/' }
+					onClick={ this.trackContactUsClick }>
+					Contact us
```

Comment:
Missing translate here. We can probably also wrap the component in Localize though let us know if there's a reason not to.